setup.py

```diff
@@ -85,6 +85,7 @@ def run(self, *args, **kwargs):
 cmake_args = ["-DPYTHON_EXECUTABLE=" + sys.executable]
 if "bdist_wheel" in sys.argv:
     cmake_args.append("-DLINK_AGAINST_PYTHON=FALSE")
+    cmake_args.append("-DNMODL_ENABLE_TESTS=FALSE")
```

Comment:
To be discussed

Reply:
Seems reasonable to me.
We already did not run the CTest suite when building the wheels.